resource-managers/mesos/src/main/scala/org/apache/spark/executor/MesosExecutorBackend.scala

```diff
@@ -48,7 +46,7 @@ private[spark] class MesosExecutorBackend
     val mesosTaskId = TaskID.newBuilder().setValue(taskId.toString).build()
     driver.sendStatusUpdate(MesosTaskStatus.newBuilder()
       .setTaskId(mesosTaskId)
-      .setState(taskStateToMesos(state))
+      .setState(MesosSchedulerBackendUtil.taskStateToMesos(state))
```

Comment:
Was this method moved to implement the TODO item? This looks unrelated to the rename changes.

Reply:
Yeah, I was in the area and noticed a TODO I could resolve quickly.

Reply:
I generally recommend not doing this because it can cause unnecessary commit conflicts.